docs/self-managed/platform-deployment/openshift.md

```diff
@@ -0,0 +1,73 @@
+---
+id: openshift
+title: "Overview"
+---
+
+A properly configured Camunda 8 platform can be deployed on OpenShift, depending primarily on the security context constraints (aka SCC) you wish to apply to it.
+
+:::note
+The recommended way to deploy Camunda 8 on OpenShift is to use the official Helm charts. These are regularly tested to ensure compatibility with a select set of OpenShift versions. You can find more about it [here](./openshift-helm.md).
+:::
+
+## Deployment
+
+Generally, if you aren't using the Helm charts, you can deploy Camunda 8 to OpenShift by following the general [Kubernetes](./kubernetes.md) guide. Then, depending on the SCC you've selected, you will have to adapt the certain resources.
+
+## What is an SCC?
```

Comment:
@npepinpe what do you think about putting an `H2` title called `Caveats` where we can use it as a unified way across platforms (also for EKS and AKS later).

so the structure would be like this:

```
## Deployment 
## Caveats
### Security Context Constraints (SCC)
#### Permissive SCC
#### Non-root SCC
#### Restrictive SCC
```

Reply:
I think the general idea is good, but I'm not sure how to make it flow here. Since the deployment section specifies that, well, how you deploy hinges entirely on which SCC you choose, it feels strange to add the `Caveat` section, which is essentially a warning section.

Reply:
My idea is that all K8s distributions (OpenShift, EKS, AKS, etc.) should work the same way ... so we have a single guide to install the charts on K8s. However, if there is anything else that should be done I'd consider it as `odd` and needs user attention.

Maybe `Caveat` is not the best word for that ... maybe "Extra steps" or something like that.

In all cases, I don't have a strong opinion about having that section but the main point here is to set the structure under some H2 section.

So this (without the `Caveat`) still stands:
```
## Deployment 
### Security Context Constraints (SCC)
#### Permissive SCC
#### Non-root SCC
#### Restrictive SCC
```

Which simply reads as "What are the things you as a user need to handle to deploy C8SM on OpenShift".

Reply:
We could rephrase `Deployment` to something more meaningful. `Required changes`? Maybe just `Configuration`?

Reply:
I'll go with simply nesting the other headers for now.

Reply:
"Configuration" sounds great :ok_hand: